packages/common/src/utils/urlFormatting.ts

```diff
@@ -132,12 +130,12 @@ export const contentListURL = (id: string, title?: string) => {
   return createPath(PATH_CONTENT_LIST, { id, title: title ? slugify(title) : undefined });
 };
 
-export const determinePath = ({ type, contentId }: { type: AppMenuType | undefined; contentId: string }) => {
+export const determinePath = ({ type, contentId, label }: { type: AppMenuType | undefined; contentId: string; label?: string }) => {
   switch (type) {
     case APP_CONFIG_ITEM_TYPE.content_list:
       return contentListURL(contentId);
     case APP_CONFIG_ITEM_TYPE.media:
-      return singleMediaURL(contentId);
+      return mediaURL({ id: contentId, title: label });
```

Comment:
Last one! We can use `label` for the `contentListURL` and `playlistURL` formatter as well 😅

Reply:
Nice catch, thank you @ChristiaanScheermeijer ❤️
I will fix it now 🚀